src/sensor/ping360.h

```diff
@@ -617,6 +622,9 @@ class Ping360 : public PingSensor {
             valid = other.transmit_duration == transmit_duration && other.gain_setting == gain_setting
                 && other.num_points == num_points && other.sample_period == sample_period
                 && other.transmit_frequency == transmit_frequency;
+            if (checkSector && valid) {
+                valid = valid && other.start_angle == start_angle && other.end_angle == end_angle;
```

Comment:
```suggestion
                valid = other.start_angle == start_angle && other.end_angle == end_angle;
```
valid must be true, it is redundant to do the comparison again.

Reply:
Good point :+1: